docs/ROADMAP.md

```diff
@@ -0,0 +1,86 @@
+# Product Roadmap
+
+MAJOR CAVEAT: This is a WORK IN PROGRESS with many items TBD. We will want weigh-in from key folks across all areas, but the idea is that this is a living, breathing document that will grow over time.
+
+### Summary
+
+There is an urgent, global need for an official WHO App to help contain and mitigate COVID-19. The app should support the WHO efforts and be available on a worldwide basis, with information tailored to the user’s location, culture and language. The app should also be sensitive to national needs and respect privacy. Once established, the system can be leveraged to proactively detect, respond early and prevent future pandemics.
+
+## Milestones
+
+Version numbering terminology:
+
+* **v0.1**: refers to the initial buildout that will be submitted to the app stores, aiming for Tuesday, March 24, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
+* **v0.2+**: any versions starting with major version v0 are not intended for public distribution or consumption
+* **v1.0**: the first version that we feel offers enough value to be published onto the app stores for public download will be v1.0
+
+### v0.1
+
+**Goal:** To get a functional build of an application that can be submitted to the Apple App Store and Google Play Store for approval. This app will _NOT_ be published into those app stores for public download, but will instead serve as an approved base for future versions and to prove out that we can deliver on the technical side. The app will serve static content manually adapted from the WHO's site in English.
+
+**Target Date:** Tuesday, March 24, 2020
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
+
+Technical Features / Requirements:
+
+* App built to be submitted on Apple App Store and Google Play Store, including code signing process
+* UI styled to exact [design spec in Figma](https://www.figma.com/file/fjzTIloCEK4FpbyDiTLj2X/iOS-UX) ([#186](https://github.com/WorldHealthOrganization/app/issues/186))
+  * Displaying static content in English only
+* App is still functional in an offline environment with default content able to be accessed without a network connection ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* Demonstrate capability to localize content from the technical platform perspective, even if other languages/cultures are _NOT_ part of that v0.1 launch
+  * Framework in place to localize UI elements ([#126](https://github.com/WorldHealthOrganization/app/pull/126) :white_check_mark:)
+  * Framework in place to load the informational content in localized formats ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* Demonstrate capability to load content dynamically from the cloud, even if this feature will _NOT_ be turned on as part of the v0.1 launch ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
```

Comment:
```suggestion
* Demonstrate capability to load content dynamically from the cloud, even if this feature will _NOT_ be turned on as part of the v0.1 launch #120
```

Reply:
Seems duplicative with line 31

Reply:
The localization and dynamic content are referring to different capabilities. Localization is referring to adapting the content to different languages/cultures, which can be done on static content pre-loaded in the app bundle. Dynamic content is referring to the ability to load content from our servers as opposed to just the default content that is pre-loaded in the app bundle, giving us the ability to update the information users see without forcing them to download an app update. Both capabilities were added in the same PR (#120) which is probably why it looks duplicative. Let me know if there is different terminology you'd like me to use to make this clearer!